testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/ejb/ssl/SSLEJBRemoteRemotingClientTestCase.java

```diff
@@ -0,0 +1,432 @@
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.PORT;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.PROTOCOL;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.REALM;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SASL_AUTHENTICATION_FACTORY;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SOCKET_BINDING;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SOCKET_BINDING_GROUP;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SSL_CONTEXT;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUBSYSTEM;
+
+/**
+ * @author <a href="mailto:tadamski@redhat.com">Tomasz Adamski</a>
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+public class SSLEJBRemoteRemotingClientTestCase {
+
+    private static final Logger log = Logger.getLogger(SSLEJBRemoteRemotingClientTestCase.class);
+    private static final String MODULE_NAME = "ssl-remote-remoting-ejb-client-test";
+    private static boolean serverConfigDone = false;
+
+    private static final String RESOURCE_PREFIX = "ejb-remote-tests";
+
+    private static final String INBOUND_SOCKET_BINDING = RESOURCE_PREFIX + "-socket-binding";
+    private static final String INBOUND_SOCKET_BINDING_TLS = RESOURCE_PREFIX + "-socket-binding-tls";
+
+    private static final int REMOTING_PORT = 4446;
+    private static final int TLS_REMOTING_PORT = 4447;
+
+    private static final String PROPERTIES_REALM = RESOURCE_PREFIX + "-properties-realm";
+    private static final String SECURITY_DOMAIN = RESOURCE_PREFIX + "-security-domain";
+    private static final String AUTHENTICATION_FACTORY = RESOURCE_PREFIX + "-sasl-authentication";
+    private static final String APPLICATION_SECURITY_DOMAIN = RESOURCE_PREFIX;
+
+    private static final String CONNECTOR = RESOURCE_PREFIX + "-connector";
+    private static final String CONNECTOR_TLS = RESOURCE_PREFIX + "-connector+tls";
+    private static final String SERVER_KEY_STORE = RESOURCE_PREFIX + "-server-key-store";
+    private static final String SERVER_KEY_MANAGER = RESOURCE_PREFIX + "-server-key-manager";
+    private static final String SERVER_TRUST_STORE = RESOURCE_PREFIX + "-server-trust-store";
+    private static final String SERVER_TRUST_MANAGER = RESOURCE_PREFIX + "-server-trust-manager";
+    private static final String SERVER_SSL_CONTEXT = RESOURCE_PREFIX + "-server-ssl-context";
+
+    private static final String KEY_STORE_KEYPASS = "clientPassword";
+
+    private static final String REMOTE = "remote";
+    private static final String REMOTE_TLS = "remote+tls";
+
+    private static final File CERTS_DIR = new File(new File("").getAbsoluteFile().getAbsolutePath()
+            + File.separatorChar + "target" + File.separatorChar + "test-classes" + File.separatorChar
+            + "ejb3" + File.separatorChar + "ssl");
```

Comment:
A more elegant solution could be using Path class instead.

Reply:
maybe I will fix it in separate PR cause now it would be imho good just to make sure that the tests are passing